deployments/boringssl-envoy-deployment.yaml

```diff
@@ -34,15 +34,15 @@ data:
         filter_chains:
           tls_context:
             common_tls_context:
-              private_key_provider:
-                provider_name: qat
-                typed_config:
-                  "@type": "type.googleapis.com/qat.QatPrivateKeyMethodConfig"
-                  section_name: ${QAT_SECTION_NAME}
-                  poll_delay: ${POLL_DELAY}
-                  private_key: "/etc/envoy/tls/tls.key"
               tls_certificates:
                 certificate_chain: { "filename": "/etc/envoy/tls/tls.crt" }
+                private_key_provider:
```

Comment:
how about we create the configmap from `examples/boringssl-envoy-conf.yaml` to avoid duplication?

Reply:
I think we should definitely take a look at the ConfigMaps anyway (see https://github.com/intel/kubernetes-qat-envoy/issues/26), but I propose we do that in a separate (Kubernetes-focused) PR.

Reply:
makes sense